addons/escoria-core/ui_library/tools/room_select/room_select.gd

```diff
@@ -54,7 +54,9 @@ func _on_button_pressed():
 	# automatic transitions. 
 	# If FORCE_LAST_SCENE_NULL is True when change_scene starts:
 	# - ESC_LAST_SCENE is set to empty
-	escoria.globals_manager.set_global("FORCE_LAST_SCENE_NULL", true, true)
+	escoria.globals_manager.set_global( \
```

Comment:
I'd better format that like

```python
escoria.globals_manager.set_global(
  escoria.room_manager.GLOBAL_FORCE_LAST_SCENE_NULL,
  true,
  true
)
```

Reply:
Will do.

Reply:
Ah, you don't need the \ in the first line. Also, please put the closing bracket into the next line all to the left aligned with the command itself.